### PR TITLE
Traces to stdout only if -fg is set

### PR DESCRIPTION
**Pull request by kzangeli**
Implements #1272 

Traces no longer sent to stdout (only logfile), unless the option **-fg** is set.

### CODE REVIEW

File: src/app/contextBroker/contextBroker.cpp [hunk 1]
```diff
@@ -69,6 +69,7 @@
 #include "parseArgs/parseArgs.h"
```
Comment (fgalan) [inline src/app/contextBroker/contextBroker.cpp]: CNR entry missing

Comment (kzangeli) [inline src/app/contextBroker/contextBroker.cpp]: Fixed in 2105139

File: src/app/contextBroker/contextBroker.cpp [hunk 2]
```diff
@@ -1481,14 +1482,22 @@ int main(int argC, char* argV[])
...
   paConfig("builtin prefix",                (void*) "ORION_");
   paConfig("usage and exit on any warning", (void*) true);
   paConfig("no preamble",                   NULL);
-  paConfig("log file time format",          (void*) "%Y-%m-%dT%H:%M:%S");
+
+
+  //
+  // If option '-fg' is set, print traces to stdout as well, otherwise, only to file
+  //
+  if (paIsSet(argC, argV, "-fg"))
```
Comment (fgalan) [inline src/app/contextBroker/contextBroker.cpp]: We should also include some comment about the new way of working in the .md doc. I can contribute to the PR with that (it is easy for me find the points to edit) but an English proof read will be welcome as usual :)

File: CHANGES_NEXT_RELEASE [hunk 3]
```diff
@@ -7,3 +7,4 @@ Fix:  Permitting attribute values to be empty in API v2 (Issue #1187 and #1188)
 Fix:  Clearer error on NGSIv2 entity attribute update, without value in payload (Issue #1164)
 Fix:  Changed  error code from 472 (Invalid Parameter) to 422 (Invalid Moficiation) in NGSIv1 in the case of missing attribute value (Issue #1164)
 Add:  Using 'count' option when getting entities (#1041)
+Fix:  Traces to stdout obly if CLI '-fg' is set (Issue #1273)
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE]: obly -> only ?

Comment (fgalan) [inline CHANGES_NEXT_RELEASE]: In addition, the issue number should be the issue one, not the PR itself.

File: src/app/contextBroker/contextBroker.cpp (hunk 2 above)
Comment (kzangeli) [inline src/app/contextBroker/contextBroker.cpp]: ok! :-)

File: CHANGES_NEXT_RELEASE (hunk 3 above)
Comment (kzangeli) [inline CHANGES_NEXT_RELEASE]: Fixed in f7046bb

File: src/app/contextBroker/contextBroker.cpp (hunk 2 above)
Comment (fgalan) [inline src/app/contextBroker/contextBroker.cpp]: Fixed in b4be9e5

**Comment by fgalan** [comment]
LGTM to Ken's part.

**Comment by kzangeli** [comment]
LGTM to all of it :-)

**Pull request merged**